openslides/core/static/js/core/site.js

```diff
@@ -86,6 +86,122 @@ angular.module('OpenSlidesApp.core.site', [
+        gettext('The 100 % base of an election result consists of');
+        gettext('Number of ballot papers (selection)');
+        gettext('Number of all delegates');
+        gettext('Number of all participants');
+        gettext('Use the following custom number');
+        gettext('Custom number of ballot papers');
+        gettext('Publish election result for elected candidates only (' +
+                'projector view)');
+        gettext('Title for PDF document (all elections)');
+        gettext('Preamble text for PDF document (all elections)');
+        //core
+        gettext('Presentation and assembly system');
+        gettext('Event name');
+        gettext('<a href="http://www.openslides.org">OpenSlides</a> is a free' +
+                ' web based presentation and assembly system for visualizing' +
+		' and controlling agenda, motions and elections of an' +
+		' assembly.');
+        gettext('General');
+        gettext('Event');
+        gettext('Short description of Event');
+        gettext('Event date');
+        gettext('Event location');
+        gettext('Event organizer');
+        gettext('Legal notice');
+        gettext('Front page title');
+        gettext('Front page text');
+        gettext('Allow access for anonymous guest users');
+        gettext('Show this text on the login page.');
+        gettext('Show logo on projector');
+        gettext('You can replace the logo. Just copy a file to ');
+        gettext('"static/img/logo-projector.png" in your OpenSlides data' + 
+		' path.');
+        gettext('Projector');
+        gettext('Show title and description of event on projector');
+        gettext('Backgroundcolor of projector header and footer');
+        gettext('Font color of projector header and footer');
+        gettext('Font color of projector headline');
+        gettext('Default countdown');
+        //Motions
+        gettext('The assembly may decide,');
+        gettext('Workflow of new motions');
+        gettext('Motions');
+        gettext('Identifier');
+        gettext('Numbered per category');
+        gettext('Serially numbered');
+        gettext('Set it manually');
+        gettext('Motion preamble');
+        gettext('Stop submitting new motions by non-staff users');
+        gettext('Allow to disable versioning');
+        gettext('Activate amendments');
+        gettext('Amendments');
+        gettext('Prefix for the identifier for amendments');
+        gettext('Number of (minimum) required supporters for a motion');
+        gettext('Choose 0 to disable the supporting system.');
+        gettext('Supporters');
+        gettext('Remove all supporters of a motion if a submitter edits his' +
+                ' motion in early state');
+        gettext('Title for PDF document (all motions)');
+        gettext('Preamble text for PDF document (all motioqns)');
+        gettext('Show paragraph numbering (only in PDF)');
+        gettext('A','Prefix for the identifier for amendments');
```

Comment:
Suggestion:
```
/// Prefix for the identifier for amendments
gettext('A');
```
"Insert context comments in JavaScript using the triple forward slash syntax of gettext." (see https://angular-gettext.rocketeer.be/dev-guide/annotate-js/)